plugins/chat/assets/javascripts/discourse/lib/chat-composer-buttons.js

```diff
@@ -111,6 +112,7 @@ export function chatComposerButtons(composer, position, context) {
       result.icon = computeButton(composer, button, "icon");
       result.disabled = computeButton(composer, button, "disabled");
       result.priority = computeButton(composer, button, "priority");
+      result.component = button.component;
```

Comment:
Same for this one... just confirming, I think you're using the `menu` service like in the topic composer

Reply:
ah yes completely forgot to remove all of these :+1: